docs/docs/adding-react-components.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Adding React Components
+---
+
+This guide covers how to add React components to your Gatsby site.
+
+## React components
+
+React components are prebuilt elements or groups of elements that can be used to prevent repeating yourself in your code.
+
+Components also offer the ability to be customised using "props" (properties). Props can be of any Javascript type, such as boolean, string, object, array or almost anything you can think of.
```

Comment:
```suggestion
Components also offer the ability to be customized using "props" (properties). Props can be of any Javascript type, such as boolean, string, object, array or almost anything you can think of.
```

For consistency, our [style guide prefers American spelling](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#prefer-us-english) of English words like this 🙂